### PR TITLE
Lambda logger/replace console logging

### PR DESCRIPTION
**Pull request by DanielAnfelt**

### CODE REVIEW

File: src/sqs-instance.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
 import AWS from 'aws-sdk';
+import config from 'config';
 
-export default new AWS.SQS({ apiVersion: '2012-11-05' });
+const api_version = config.get('aws.api_version');
+export default new AWS.SQS({ api_version: api_version });
```
Comment (DanielAnfelt) [inline src/sqs-instance.js]: Very unsure - is this the correct way of referencing the value, or can one just insert the const without brackets or key? 

i.e. …AWS.SQS(api_version);

Comment (mferno) [inline src/sqs-instance.js]: You could `export default new AWS.SQS({ api_version });` but this is fine!

File: index.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
+import logger from './src/util/logger';
```
Comment (mferno) [inline index.js]: Use `require` instead of `import` since the index file has to be ES5

Comment (mferno) [inline index.js]: Also we try to avoid the `util` pattern so it's better to put it plain in the src folder

Comment (mferno) [inline index.js]: The logger file I mean

File: src/sqs-redriver.js [hunk 3]
```diff
@@ -119,7 +120,7 @@ const api = {
         .then(extractQueueUrls)
         .then(fetchMessagesUntilEmpty)
         .catch(err => {
-          console.warn('redriveMessages failed: %s', err);
+          logger.trace('redriveMessages failed: %s', err);
```
Comment (mferno) [inline src/sqs-redriver.js]: Logging level warn ?

File: index.js [hunk 4]
```diff
@@ -1,14 +1,24 @@
+const logger = require('./src/logger');
```
Comment (mferno) [inline index.js]: You need to require the file after the NODE_ENV has been set. So inside the handler.

File: index.js [hunk 5]
```diff
@@ -1,14 +1,24 @@
...
 const redriver = require('./build/sqs-redriver');
 
-exports.handler = function(sns_event) {
-  console.log('Starting queue redrive...');
+exports.handler = function(sns_event, context, callback) {
+  if (context && context.functionName === 'ReDriveQueueMessages') {
+    process.env.NODE_ENV = 'production';
+    process.env.NODE_LOG_FILE_PATH = '/dev/null';
```
Comment (mferno) [inline index.js]: Remove

File: index.js [hunk 6]
```diff
@@ -1,14 +1,24 @@
...
+    context.callbackWaitsForEmptyEventLoop = false;
+  }
+  logger.info('Starting queue redrive...');
 
   redriver.default.redriveMessages({ sns_event: sns_event })
       .then(() => {
-        console.log('Queue redrive complete');
+        logger.info('Queue redrive complete');
+        setImmediate(callback);
```
Comment (mferno) [inline index.js]: callback();

File: index.js [hunk 7]
```diff
@@ -1,14 +1,24 @@
...
       })
       .catch(err => {
-        console.log('Queue redrive failed', err);
+        logger.trace('Queue redrive failed', err);
+        setImmediate(() => {
```
Comment (mferno) [inline index.js]: callback(err);

**Pull request merged**